src/envs/cover.py

```diff
@@ -706,11 +805,16 @@ def _get_hand_regions(self, state: State) -> List[Tuple[float, float]]:
 
     @staticmethod
     def _Covers_holds(state: State, objects: Sequence[Object]) -> bool:
-        # Overriding because of the change from "pose" to "x".
+        # Overriding because of the change from "pose" to "x" and because
+        # blocks x position is updated in every step of simulate and not just at
```

Comment:
this reads like "blocks cross position" -- do you mean "the block's x-position"?